src/controller/python/chip/setup_payload/setup_payload.py

```diff
@@ -66,15 +66,21 @@ def ParseManualPairingCode(self, manualPairingCode: str):
 
         return self
 
-    # DEPRECATED
```

Comment:
This is deprecated. What is being attempted here? "PrintOnboardingCodes" is not compatible with custom PAKE verifier computation

Reply:
Use the wrong passcode according to the use case requirements, and the Python side currently does not have an interface to generate SetUpManualCode through the passcode

Reply:
I found the Python script that generates SetUpCode in the source code(`src/setup_payload/python/generate_setup_payload.py`), I will close this PR